apps/DisplayCluster/resources/MasterContentWindow.qml

```diff
@@ -27,6 +27,26 @@ BaseContentWindow {
             displaygroup.focus(contentwindow.id)
     }
 
+    Item {
```

Comment:
Does this have to be a separate item? Can't the functions Keys.on* be added to contentInteractionArea?